rust/kernel/pm.rs

```diff
@@ -116,3 +116,73 @@ unsafe impl<T: PointerWrapper> Sync for NoOperations<T> {}
 
 // SAFETY: `NoOperation` provides no functionality, it is safe to send it to different threads.
 unsafe impl<T: PointerWrapper> Send for NoOperations<T> {}
+
+/// Device run-time power management helper functions.
+///
+/// C header: [`include/linux/pm_runtime.h`](../../../../include/linux/pm_runtime.h)
+pub mod runtime {
```

Comment:
Consider moving this into its own `rust/kernel/pm/runtime.rs` (I assume it will grow over time).

Reply:
Sure, thanks.